voter-dapp/src/containers/VoteData.js

```diff
@@ -87,24 +87,29 @@ function useVoteData() {
           }
         }
 
+        // Data on unique users:
+        const uniqueCommits = Object.keys(uniqueVotersCommitted).length;
+        const uniqueReveals = Object.keys(uniqueVotersRevealed).length;
+        const uniqueClaimers = Object.keys(uniqueRewardClaimers).length;
+        const uniqueRevealsPctOfCommits = uniqueCommits > 0 ? (100 * uniqueReveals) / uniqueCommits : 0;
+        const uniqueClaimersPctOfReveals = uniqueReveals > 0 ? (100 * uniqueClaimers) / uniqueReveals : 0;
+
         // Insert round data into new object.
         newVoteData[newRoundKey] = {
           totalSupplyAtSnapshot: dataForRequest.totalSupplyAtSnapshot,
-          uniqueCommits: Object.keys(uniqueVotersCommitted).length,
+          uniqueCommits: uniqueCommits.toString(),
```

Comment:
However, I do force all properties of the returned object to be of type `string | null` here. This way the front end only needs to worry about displaying data